src/cow-react/modules/limitOrders/pure/ReceiptModal/CurrencyField.tsx

```diff
@@ -11,13 +11,13 @@ interface Props {
 
 export function CurrencyField({ label, token, amount }: Props) {
   return (
-    <styledEl.CurrencyField title={amount?.toExact()} border="rounded-full">
-      <styledEl.FieldTitle>{label}</styledEl.FieldTitle>
+    <styledEl.CurrencyField title={amount?.toExact()}>
+      <b>{label}</b>
 
```

Comment:
Basically reducing the amount of required styled components, keeping things more self contained where possible/allowable.